src/components/text/text.module.css

```diff
@@ -22,32 +19,33 @@
 .h1 {
   font-size: 4.5rem;
   font-style: normal;
-  font-weight: 600;
-  line-height: 120%;
+  font-weight: 450;
+  line-height: 111.111%;
 }
 
 .h2 {
-  font-size: 3rem;
-  font-weight: 600;
-  line-height: 120%;
+  font-size: 3.5rem;
+  font-weight: 450;
+  line-height: 114.286%;
 }
 
 .h3 {
-  font-size: 2.125rem;
-  font-weight: 600;
+  font-size: 2rem;
+  font-weight: 450;
+  line-height: 125%;
 }
 
 .h4 {
   font-size: 1.5rem;
   font-style: normal;
-  font-weight: 500;
-  line-height: 120%; /* 1.8rem */
+  font-weight: 450;
```

Comment:
Have you verified that this actually works as expected? Meaning, that the font included is actually a variable font and supports all types of weight (as this is typically not included in most font types)

Reply:
Because I think this should actually use `font-variation-settings: "wght" 450;` 🤔

Reply:
Yup, I'd say it works as expected! For instance, this is weight 450 vs 550:

#### 450
<img width="494" alt="Screenshot 2024-12-06 at 12 41 10" src="https://github.com/user-attachments/assets/89313947-a580-40c4-b3ad-7b5231d89b7f">

#### 550
<img width="513" alt="Screenshot 2024-12-06 at 12 41 03" src="https://github.com/user-attachments/assets/0aa80c5b-8daa-4ad2-a91e-cfaa71c5f51f">

Reply:
But I think 450 will here show 400, not actually 450. What happens if you replace this line and compare 400 vs 450?:


```suggestion
  font-variation-settings: "wght" 450; 
```
Variable fonts are odd like this. They have settings inside them self that is other than the weight from CSS, so using font-variation-settings instructs the font it self to change.

Reply:
With `font-variation-settings: "wght" 450;`:
<img width="493" alt="Screenshot 2024-12-06 at 12 46 59" src="https://github.com/user-attachments/assets/dd7d46af-0667-4765-ace3-f1ef8b76bd2e">

vs
`font-weight: 450;`:
<img width="494" alt="Screenshot 2024-12-06 at 12 41 10" src="https://github.com/user-attachments/assets/c79fac0f-205f-4277-b3df-296d5f87e241">

Reply:
Impossible for me to see the difference, but that might be because the difference is small also 😅

Reply:
Haha yeah, they look quite identical to me

Reply:
If we can't see the difference, I guess we can just go with using `font-weight`?

Reply:
Lets just go with it. I'll experiment on my own just because I can't let things go in knowing the details and finding out if it works the way I think it works, but that is my torch to carry on my own.

Reply:
cross to carry?

Reply:
cross to bear, I believe!

Reply:
> Lets just go with it. I'll experiment on my own just because I can't let things go in knowing the details and finding out if it works the way I think it works, but that is my torch to carry on my own.

Hehehe, keep me posted on your findings!

Reply:
![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExb201M3NqYzZudzlrbzRuYXJ0cndlMWttczhlYzY3MXB3a3hzbWF1bSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/1etWxo9VMHrjtyz4Uj/giphy.gif)